### DISCUSSION TITLE
Using pg_dump over ssh

### DISCUSSION THREAD

**Discussion by omidraha** [prodrigestivill/docker-postgres-backup-local#110] [Ideas]
Hi,
This is my [approach](https://github.com/omidraha/periodic-pgdump-over-ssh) around your great project.

**Comment by prodrigestivill** [comment]
Nice!

Also to archive this it can also be done by configuring a docker-compose together with local backup and a [ssh tunnel](https://hub.docker.com/search?q=ssh%20tunnel&type=image) image and using the redirected ports.
